paperqa/clients/crossref.py

```diff
@@ -38,7 +38,9 @@
 
 CROSSREF_HOST = "api.crossref.org"
 CROSSREF_BASE_URL = f"https://{CROSSREF_HOST}"
-CROSSREF_API_REQUEST_TIMEOUT = 5.0
+CROSSREF_API_REQUEST_TIMEOUT = float(
+    os.environ.get("CROSSREF_API_REQUEST_TIMEOUT", "10.0")
```

Comment:
I like the configurability, but why bump from 5-sec to 10-sec default here? Isn't that going to make things worse?